less/index.less

```diff
@@ -54,6 +54,320 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+}
+// Variables
+@primary-color: #4C86A8;
+@black: #00241B;
+@white: #F3F8F2; 
+@primary-light: lighten(@primary-color, 45%);
+@base-size: 1.6rem;
+// Mixins
+.header-font(){
+  font-family: 'Eczar', serif;
+}
+.base-font(){
+  font-family: 'Gentium Basic', serif;
+}
+.text-center(){
+  text-align: center;
+}
+.content-border(){
+	border: 1px red solid;
+} 
+.layout(@j-content, @a-items){
+	display: flex;
+	justify-content: @j-content;
+	align-items: @a-items;
+}
+.cards(){
+  background: white;
+  h3{
+    .header-font();
+    font-weight: semi-bold;
+    font-size: 3rem;
+    padding: 10px 0;
+  }
+  img{
+    border-top: 3px solid @black;
+    border-bottom: 3px solid @black;
+    @media (max-width: 500px) {
+      width:100%;
+    }
+  }
+  p{
+    .base-font();
+    padding: 10px 0;
+    font-size: 2rem;
+  }
+  text-align: center;
+  border: 3px solid @black;
+  border-radius: 15px;
+  color: @black;
+  @media (max-width:800px) {
+    width: 40%;
+  }
+  @media (max-width: 500px) {
+    width:100%;
+  }
+}
+.h2(@size, @margin){
+ font-size: @size;
+ margin: @margin;
+}
+.background-image(@url){
+  background-image: url(@url);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+// General Styles
+html{
+	font-size: 62.5%;
+}
+body{
+  background: @primary-light;
+  color: @black;
+}
+img{
+	max-width: 100%;
+	height: auto;
+}
+.container{
+	font-size: @base-size;
+	margin: 0 auto;
+	max-width: 1100px;
+}
+// Page Styles
+// Navigation 
+header {
+  .layout(space-between, baseline);
+  .header-font();
+  @media (max-width: 600px) {
+    flex-direction: column;
+    width: 100%;
+   .layout(center, center);
+  }
+	 background: @primary-color;
+	 padding: 20px 0;
+ h2{ 
+    font-size: 3rem;
+		padding-left: 3%;
+		width: 50%;
+    color: @white;
+    @media (max-width: 600px) {
+      .text-center();
+      padding-left: 0;
+    }
+	}
+nav{
+	.layout(space-around, center);
+  width: 50%;
+  @media (max-width: 600px) {
+    width: 100%
+  }
+	a{
+		text-decoration: none;
+		color: @white;
+		font-size: 2rem;
+  }
+  @media (max-width: 600px) {
+    padding-top: 20px;
+  }
+}
+}
+// Main Header
+.main-header{
+  .background-image("../img/background.jpg");
+  padding-top:40px;
+  margin: 0px auto 40px;
+  border-bottom: 3px solid @black;
+  @media (max-width: 600px) {
+    flex-wrap: wrap;
+    border-bottom: none;
+  }
+	.column{
```

Comment:
This could use a little tidying. The styles don't all line up making it a little hard to read.